examples/picorv32_ram/sram/sky130_sram_2k.py

```diff
@@ -0,0 +1,17 @@
+import os
+import siliconcompiler
+
+def setup(chip):
+    # Core values.
+    design = 'sky130_sram_2k'
+    stackup = chip.get('option', 'stackup')
+
+    # Create library Chip object.
+    lib = siliconcompiler.Chip(design)
+    libdir = os.path.dirname(__file__)
+    lib.set('output', stackup, 'gds', f'{libdir}/sky130_sram_2kbyte_1rw1r_32x512_8.gds')
```

Comment:
It could go in a new `third_party/pdks/skywater/skywater130/libs/sky130ram/` directory. We tried to get rid of submodules as part of the recent refactor, so that's probably not a great option. We might be able to gzip it.

Reply:
gzip works natively with KLayout, and brings it down to 4.5MB.

Still not ideal, but asking people to run the build scripts would mean they'd have to install OpenRAM to run the example.

Reply:
I think the GDS locally is fine for now so as to not block progress. At some point, we definitely want to pull out all technology locked files out of SC into a separate repo.